ooniprobe/Utility/TestUtility.mm

```diff
@@ -144,7 +144,23 @@ + (void)downloadUrlsCallback:(NSData *)data
     successcb(urls);
 }
 
-+ (void)removeFile:(NSString*)fileName {
++ (void)deleteUploadedJsons:(void (^)(Measurement*))successcb onError:(void (^)(NSError*))errorcb {
+    for (Measurement *measurement in [Measurement measurementsWithJson]){
+        [measurement getExplorerUrl:^(NSString *measurement_url){
+            [TestUtility removeFile:[measurement getReportFile]];
+            successcb(measurement);
+        } onError:^(NSError *error){
+            errorcb(error);
+        }];
+    }
+}
+
+//TODO where and when to call it?
++ (void)deleteUploadedJsons{
+    [self deleteUploadedJsons:nil onError:nil];
+}
+
++ (BOOL)removeFile:(NSString*)fileName {
```

Comment:
It seems we're not ever using the return value, so we probably don't need to propagate it? (Or am I missing something?)

Reply:
I added it in case we wanted to do a unit test only for removeFile